repo_nginx_fancyindex/footer.html

```diff
@@ -46,6 +46,22 @@ <h2><span>Post</span>-installation</h2>
 
       <p>À partir de l'administration web de YunoHost, vous pouvez désormais installer les apps <em>VPN Client</em> et <em>Wifi Hotspot</em>, puis aller les configurer.
     </section>
+
+    <section id="install-with-hypercube">
+        <h2><span>Hyper</span>cube</h2>
+
+        <p>La méthode d'installation recommandée consiste à utiliser un fichier hypercube généré via <a href="https://install.labriqueinter.net/">cette page</a>.</p>
+
+        <p>Le fichier hypercube peut être fourni au script <code>install-sd.sh</code> de la manière suivante :</p>
+
+        <pre>% bash install-sd.sh -y fichier_hypercube.hyp</pre>
```

Comment:
I think the file generated by `install.labriqueinter.net` is called `install.hypercube` .